src/meta.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_META_H
+#define EP_META_H
+
+// Headers
+#include <map>
+#include <memory>
+#include <string>
+#include "filefinder.h"
+#include "inireader.h"
+
+
+/**
+ * The Meta class loads from a file (easyrpg.ini) and contains 
+ * additional information on a per-game basis.
+ */
+
+class Meta {
+public:
+	// @ini_file - The path to the ini file to load
```

Comment:
When making doc-comments please follow the javadoc-style we use:

```
/**
 * brief description
 * @param ini_file The path to the ini file to load
 * @return some return
 */
```
see other files for examples

btw, the argument is called meta\_file, not ini\_file